File: tests/fixtures/vulnerable_app.go
// Fixture: Go app using quantum-vulnerable crypto.
package main

import (
	"crypto/ecdsa"
	"crypto/elliptic"
	"crypto/rand"
	"crypto/rsa"
)

func main() {
	_, _ = rsa.GenerateKey(rand.Reader, 2048)
	_, _ = ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
}

File: tests/fixtures/vulnerable_app.js
// Fixture: Node.js app using quantum-vulnerable + broken crypto.
const crypto = require('crypto');

const { publicKey, privateKey } = crypto.generateKeyPairSync('rsa', {
  modulusLength: 2048,
});

const md5sum = crypto.createHash('md5').update('hi').digest('hex');
const sha1sum = crypto.createHash('sha1').update('hi').digest('hex');

const ecdh = crypto.createECDH('secp256k1');
